### PR TITLE
document private transaction lifecycle

### LINKED ISSUE #143: Add flow of processing private transactions
Add descriptive flows and maybe diagrams for processing private transactions.

The following content is removed with PR #139 and can be referenced for this PR:
- [Private transaction lifecycle](https://docs.goquorum.consensys.net/en/21.7.1/Concepts/Privacy/PrivateTransactionLifecycle/)
- [Privacy marker transaction flows](https://docs.goquorum.consensys.net/en/21.7.1/Concepts/Privacy/PrivacyMarkerTransactions/#flows)

Also reference Besu's [Processing private transactions](https://besu.hyperledger.org/en/stable/Concepts/Privacy/Private-Transaction-Processing/).

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull Request Description

Document lifecycles of private transactions and PMTs, including new diagrams.

## Fixed Issue(s)

fixes #143 

## Preview

https://pegasys-docgoquorum--158.com.readthedocs.build/en/158/Concepts/Privacy/PrivateTransactionLifecycle/

### CODE REVIEW

File: docs/Concepts/Privacy/PrivateTransactionLifecycle.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+This topic illustrates the high-level lifecycle of normal [private transactions](PrivateAndPublic.md#private-transactions)
+and [privacy marker transactions (PMTs)](PrivacyMarkerTransactions.md).
+
+In both examples, participants A and B are party to the transaction and participant C is not.
+
+## Normal private transactions
+
+![Private transaction flow](../../images/PrivateTxnFlow.png)
+
+1. Participant A sends a private transaction to GoQuorum.
```
Comment (antonydenyer) [inline docs/Concepts/Privacy/PrivateTransactionLifecycle.md:16]: Participant A sends a private transaction to their GoQuorum node.

File: docs/Concepts/Privacy/PrivateTransactionLifecycle.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+1. GoQuorum serializes the private transaction `data` value to JSON and sends it to Tessera with participant details.
```
Comment (antonydenyer) [inline docs/Concepts/Privacy/PrivateTransactionLifecycle.md:17]: to their Tessera with

File: docs/Concepts/Privacy/PrivateTransactionLifecycle.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+   GoQuorum creates a public PMT with the `data` field set to the hash.
+   GoQuorum signs the PMT.
+1. GoQuorum distributes the PMT throughout the network.
+   All nodes can see the PMT.
+
+    The PMT is processed in the same way as a standard public transaction.
+    Since the PMT's `to` value is the address of the privacy precompile contract, that contract is called.
+
+1. GoQuorum calls the privacy precompile with the PMT `data`.
+1. GoQuorum attempts to retrieve the decrypted private transaction from Tessera.
```
Comment (antonydenyer) [inline docs/Concepts/Privacy/PrivateTransactionLifecycle.md:43]: from Tessera using the publicly available hash from the PMT

**Review by antonydenyer — APPROVED** [review]
A couple of comments overall looks good

**Review by rolandtyler — APPROVED** [review]
lgtm

**Pull request merged**